src/components/Reporta.tsx

```diff
@@ -27,11 +30,7 @@ export default function ReportaForm() {
           </h4>
         </div>
 
-        <form
-          id="formSheet"
-          className={styles.formSheet}
-          onSubmit={(e) => Submit(e)}
-        >
+        <form id="formSheet" className={styles.formSheet} onSubmit={submit}>
```

Comment:
any reason why we move frome the onSubmit={Submit} to onSubmit={(e) => Submit(e)} ?

Reply:
Saves a function declaration, which would defeat the purpose of the useCallback

Reply:
isn't (e) => Submit(e) a function ?